server/command.go

```diff
@@ -392,3 +394,35 @@ func (c *conn) handleReportSplit(req *pdpb.Request) (*pdpb.Response, error) {
 		ReportSplit: split,
 	}, nil
 }
+
+func (c *conn) handleGetPDMembers(req *pdpb.Request) (*pdpb.Response, error) {
```

Comment:
This is almost the same as the members API, you can combine them together and export to API.